packages/swap/.env.backspin

```diff
@@ -0,0 +1,17 @@
+#/-------------------[DOTENV_PUBLIC_KEY]--------------------/
+#/            public-key encryption for .env files          /
+#/       [how it works](https://dotenvx.com/encryption)     /
+#/----------------------------------------------------------/
+DOTENV_PUBLIC_KEY_BACKSPIN="03849c664675501e7de2cc693d3cbcb30076b631029439d898fe85d454c0d75d40"
+
+# .env.backspin
+CHAINFLIP_NETWORK=backspin
+RPC_NODE_HTTP_URL=encrypted:BNt54eOzxyfXQIgtXHWt0zWs9Zd2d3/7tiCls96E7BfWtVNbaBl98914mzNSiC9jnXSlL86Yf7U+o0gwzbIt61cKx1AoBO6eYfhRcromw0C8RSO/Bpw4PYOKNx/B/TTiPKg46n5rJDljqGXEYKiUPzH+hqJRO1fGXNrgUPjTfoyPg/8+SMo=
```

Comment:
encrypted all the publicly accessible urls in these files because im paranoid about random github scrapers bombing them with requests